docs/README.md

```diff
@@ -1,4 +1,45 @@
-# Dashboard Documentation
+## Advanced Topics and Tutorials
```

Comment:
Might be nit-picky but I wouldn't consider the FAQ an advanced topic. Maybe title it "Additional topics and documentation"?

Reply:
Not nit-picky at all, I took a lot of design inspiration from https://github.com/kubernetes/minikube but I agree that FAQ doesn't fall under advanced :) - I'll make those changes.